test/canary/resources/canary_config.json

```diff
@@ -1,9 +1,28 @@
 {
   "metrics": {
```

Comment:
Need to include aggregate dimensions so we can get a metric without dimensions. Looking at the canary metrics from its brief run, it adds a metric with a couple dimensions including the host IP, which makes it difficult to use.

Reply:
Added